app/models/item.rb

```diff
@@ -1,3 +1,5 @@
 class Item < ApplicationRecord
-  
+  belongs_to :merchant
+  has_many :invoice_items
+  # has_many :invoices, through: invoice_items
```

Comment:
Should this line be uncommented?

Reply:
It was throwing me an error, still have to figure out why!